dotnet/src/SemanticKernel.Abstractions/Contents/StreamingChatMessageContent.cs

```diff
@@ -17,9 +18,52 @@ namespace Microsoft.SemanticKernel;
 public class StreamingChatMessageContent : StreamingKernelContent
 {
     /// <summary>
-    /// Text associated to the message payload
+    /// A convenience property to get or set the text of the first item in the <see cref="Items" /> collection of <see cref="StreamingTextContent"/> type.
     /// </summary>
-    public string? Content { get; set; }
+    public string? Content
+    {
+        get
+        {
+            var textContent = this.Items.OfType<StreamingTextContent>().FirstOrDefault();
+            return textContent?.Text;
+        }
+        set
+        {
+            if (value is null)
+            {
+                return;
```

Comment:
Getting returns the first item but setting to null does nothing, is that what we want?


If I had only one item in the list should we remove it?

Reply:
> If I had only one item in the list should we remove it?

Good catch. I fixed it slightly differently. I don't remove the item but just update the first item's text/content to null if null is assigned to the property so both the setter and getter have consistent behavior. I'll also register a task to handle a similar case in the `ChatMessageContent` class.